main.tex

```diff
@@ -161,10 +175,11 @@ \subsection{Convergence Analysis}
 This is sometimes stated more weakly by saying simply that $\| \vec u_h - R_hu \|_{\infty,h}$ is $\mathcal O(h^k)$. 
 \end{definition}
 
-\begin{theorem}
-    Consider the previously defined problem and its finite difference discretization. If the scheme is consistent, then stability and convergence are equivalent. 
```

Comment:
Agrega la siguiente frase: This celebrated result, known as the Lax equivalence theorem, has its counterpart for multistep methods and is known as the Dahlquist equivalence theorem. We do not provide any details on the latter.